puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/foreman-nightly-rpm-pipeline.yaml

```diff
@@ -10,6 +10,7 @@
     dsl:
       !include-raw:
         - pipelines/release/foremanRelease.groovy
+        - pipelines/test/katelloDevel.groovy
```

Comment:
@johnpmitsch This is now how we want to do this. I think we still want this to be a distinct job, that is triggered by `katello-nightly-rpm-pipeline`. Here is an example:

https://github.com/theforeman/foreman-infra/blob/ea569135ebfdaf2d5e08e8c6918a3ce2741130a1/puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/katello-nightly-rpm-pipeline.yaml#L4

This is a reverse trigger that says "I should be triggered when job X completes"